### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by sommeeeer**
Just some minor typos in the docs

### CODE REVIEW

File: docs/pages/common_issues/isr.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ import { CloudFrontClient, CreateInvalidationCommand } from "@aws-sdk/client-clo
 
 const cloudFront = new CloudFrontClient({});
 
-async function invalidateCFPaths(paths: string[]) {
+async function invalidateCloudFrontPaths(paths: string[]) {
```
Comment (khuezy) [inline docs/pages/common_issues/isr.mdx:33]: Can you also update the reference to the function below:
```
// This costs $0.005 x 3 = $0.015 after the first 1000 paths
await invalidateCFPaths(["/page/a", "/page/b", "/page/c"]);
// This costs $0.005, but also invalidates other routes such as "page/d"
await invalidateCFPaths(["/page/*"]);
```

Comment (sommeeeer) [reply docs/pages/common_issues/isr.mdx:33]: Yup will do!

**Review by khuezy — APPROVED** [review]
LGTM thanks!

File: docs/pages/common_issues/_meta.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (khuezy) [inline docs/pages/common_issues/_meta.json:1]: I'm not sure what this _meta.json file does.

Comment (sommeeeer) [reply docs/pages/common_issues/_meta.json:1]: Its for making the Isr part uppercase in this menu:
image

**Pull request merged**